package.json

```diff
@@ -30,14 +30,16 @@
   },
   "private": false,
   "dependencies": {
-    "@xmldom/xmldom": "^0.8.4",
+    "@xmldom/xmldom": "^0.8.10",
+    "apigeelint": "^2.55.4",
```

Comment:
why is apigeelint added to the main package.json?

Reply:
shouldn't be!